src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1041,11 +1037,23 @@ public LocalDate fromString(String string) {
           checkBoxCvrTreatBlankAsUndeclaredWriteIn
               .setSelected(ContestConfig.SUGGESTED_TREAT_BLANK_AS_UNDECLARED_WRITE_IN);
         }
+        case CSV -> {
+          buttonAddCvrFile.setDisable(false);
+          textFieldCvrFilePath.setDisable(false);
+          buttonCvrFilePath.setDisable(false);
+          textFieldCvrContestId.setDisable(true);
+          textFieldCvrFirstVoteCol.setDisable(false);
+          textFieldCvrFirstVoteCol
+                  .setText(String.valueOf(ContestConfig.SUGGESTED_CVR_FIRST_VOTE_COLUMN));
+          textFieldCvrFirstVoteRow.setDisable(false);
+          textFieldCvrFirstVoteRow
+                  .setText(String.valueOf(ContestConfig.SUGGESTED_CVR_FIRST_VOTE_ROW));
+        }
         case CLEAR_BALLOT, DOMINION, HART -> {
           buttonAddCvrFile.setDisable(false);
           textFieldCvrFilePath.setDisable(false);
           buttonCvrFilePath.setDisable(false);
-          textFieldCvrContestId.setDisable(false);
+          textFieldCvrContestId.setDisable(true);
```

Comment:
Curious what's motivating this change? Was this a bug in there before? If so, how did you come to that conclusion?

Finally, if it is the case that this should be changed, just remove this line instead since the default is for it to be disabled.

Reply:
Good catch -- this was a mistake in the latest commit, @tarheel requested changing this a few lines above. Will revert.